### ISSUE TITLE
A little help needed + some issues

### ISSUE THREAD

**Issue by joveice** [BadChoice/handesk#163]
What I wanted to do:

- [x] REPLY: Send from Support@mydomain as username Support (hard coded as it seems to ignore MAIL_FROM_ADDRESS  and MAIL_FROM_NAME config)
- [x] FIRST CONTACT: Send from Support@mydomain as username Support
- [x] Don't send email to the admin when there are updates to a ticket which isn't assigned to him
- [ ] Only pick emails sent to a specific alias (etc if you have aliases on gmail to save money for multiple addresses, account, support, contact etc)
Where can I do whats left on the list? I didn't quite understand the code structure.

Issues:
When in a ticket you can edit the client from address and name, changing that does not seem to do anything or save it.
Seems like I get notified when commenting on my own ticket as a agent?
Maby the reply to this email or click on the link is useless as the client doesn't have a account for the website anyways? (I removed this from mine)

Love the project!

**Comment by BadChoice** [comment]
· You can change whole email configuration in both .env and app/email.php

· To pick emails only from one alias... this is not possible right now, as it fetches a full mailbox, but sounds like a good feature to add

· About admin notifications, there is a setting to turn off different kinds of notifications, it might help?

You shouldn't get notified yourself when commenting a ticket, unless the requester is you too

**Comment by joveice** [comment]
@BadChoice 

1. Still ignores it as it seems to be hardcoded to behave differently. ([L61](https://github.com/BadChoice/handesk/blob/dev/app/Notifications/NewComment.php#L61))
2. No way to filter out while collecting?
3. Where can I find these settings?

**Comment by joveice** [comment]
Okey I solved more stuff. I also see that it sends a email to the client from them self with the address specified in fetch. (I simply hard coded 'Support', but that probably won't fit everyone)
The Powered by logo is located at "handesk.dev" which is hard coded (could probably do a PR).
I found the notification settings and it now works as expected.
If you save on the profile page with a email currently in use you trigger a exception due to it trying to save as a duplicate on a unique column.
I will probably find more stuff, and I could probably also start doing PR's when I manage to set myself into the code layout. (I'm currently developing with Php and Laravel in the daily).
